frontend/public/components/operator-lifecycle-manager/package-manifest.tsx

```diff
@@ -85,8 +99,8 @@ export const PackageManifestList = requireOperatorGroup((props: PackageManifestL
               .filter(og => _.isEmpty(props.namespace) || og.metadata.namespace === props.namespace)
               .some(og => supports(installModesFor(rowProps.obj)(defaultChannelFor(rowProps.obj)))(og))}
           defaultNS={_.get(props.operatorGroup, 'data[0].metadata.namespace')} />}
-        label="Package Manifests"
-        EmptyMsg={() => <MsgBox title="No PackageManifests Found" detail="The catalog author has not added any packages." />} />
+        EmptyMsg={() => <MsgBox title="No PackageManifests Found" detail="The catalog author has not added any packages." />}
+        virtualize />
```

Comment:
Previously we disabled virtualization in this table. Is it safe to add it back?

Reply:
Should be. This view will be removed soon anyway.